From efc5777e49c0c80c744c48b1334c9c16f9f264a6 Mon Sep 17 00:00:00 2001
From: Steev Klimaszewski <steev@gentoo.org>
Date: Sun, 19 Aug 2012 03:58:15 -0500
Subject: [PATCH] Add search for hypot

When linking with gold, link fails because hypot isn't found.

Signed-off-by: Steev Klimaszewski <steev@gentoo.org>
---
 configure.ac |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index 64dd3ad..621e295 100644
--- a/configure.ac
+++ b/configure.ac
@@ -163,6 +163,7 @@ dnl Check dependencies
 # SHELL_CFLAGS     for shell implementation.
 # SHELL_LIBS
 
+AC_SEARCH_LIBS([hypot],[m])
 
 PKG_CHECK_MODULES(LIBDOCUMENT, gtk+-$GTK_API_VERSION >= $GTK_REQUIRED gio-2.0 >= $GLIB_REQUIRED)
 PKG_CHECK_MODULES(LIBVIEW, gtk+-$GTK_API_VERSION >= $GTK_REQUIRED gail$GAIL_API_VERSION >= $GTK_REQUIRED gthread-2.0 gio-2.0 >= $GLIB_REQUIRED)
-- 
1.7.3.4

